src/phoenix/experimental/evals/functions/classify.py

```diff
@@ -85,31 +86,36 @@ def llm_classify(
         and model.supports_function_calling
     )
 
-    # TODO: support explanation without function calling
-    if provide_explanation and not use_openai_function_call:
-        raise ValueError(
-            "explanation is not currently available for models without OpenAI function calling"
-        )
-
     model_kwargs: Dict[str, Any] = {}
     if use_openai_function_call:
         openai_function = _default_openai_function(rails, provide_explanation)
         model_kwargs["functions"] = [openai_function]
         model_kwargs["function_call"] = {"name": openai_function["name"]}
 
-    eval_template = normalize_template(template)
-    prompts = map_template(dataframe, eval_template)
+    eval_template = normalize_classification_template(rails=rails, template=template)
+
+    prompt_options = PromptOptions(provide_explanation=provide_explanation)
+    prompts = map_template(dataframe, eval_template, options=prompt_options)
+
     labels: List[str] = []
     explanations: List[Optional[str]] = []
+
     if generation_info := model.verbose_generation_info():
         printif(verbose, generation_info)
 
     for prompt in tqdm(prompts):
         with set_verbosity(model, verbose) as verbose_model:
             response = verbose_model(prompt, instruction=system_instruction, **model_kwargs)
         if not use_openai_function_call:
-            unrailed_label = response
-            explanation = None
+            if provide_explanation:
+                unrailed_label, explanation = eval_template.parse_label(response), response
+                printif(
+                    verbose and unrailed_label == NOT_PARSABLE,
+                    f"- Could not parse {repr(response)}",
+                )
```

Comment:
documentation: I would add a bit more color here so the user understands what part of the execution is failing. Including maybe the response